.changes/v3.8.0/902-notes.md

```diff
@@ -0,0 +1,8 @@
+* Bumps Go to 1.18 in `go.mod` as the minimum required version. [GH-902]
+* Does `make fmt` which adjusts doc formatting for Go 1.19 [GH-902]
+* Adjusts GitHub actions to use the latest code [GH-902]
```

Comment:
```suggestion
* Adjust GitHub actions to use the latest code [GH-902]
```
But this raises an interesting question: how to procede with changelog entries which are not very actual to end-users. But we can amend them during pre-release changelog review.

Reply:
Adjusted. Well, I am putting it in `notes` section which is probably the best we can get. In the end we might skip it at all during final release if we decide to.